src/theme/common.scss

```diff
@@ -433,3 +433,44 @@ strong {
 .follow-up-converted .mat-progress-bar-fill::after {
   background-color: mat.get-color-from-palette($sormas-alert, 300);
 }
+
+.follow-up-cooperative .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-success, 600);
+}
+
+.follow-up-uncooperative .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-alert, 700);
+}
+
+.follow-up-unavailable .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-warning, 700);
+}
+
+.follow-up-not-visited .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-accent, 500);
+}
+
+.visits-cooperative .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-success, 600);
+}
+
+.visits-uncooperative .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-alert, 700);
+}
+
+.visits-unavailable .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-warning, 700);
+}
+
+.visits-missed .mat-progress-bar-fill::after {
+  background-color: mat.get-color-from-palette($sormas-warning, 800);
+}
+
+mat-tooltip-component {
+  .mat-tooltip {
+    color: mat.get-color-from-palette($sormas-accent, 800);
+    @include mat.elevation(2);
+    background-color: mat.get-color-from-palette($sormas-primary, 50) !important;
```

Comment:
please find a solution so we do not use !important .... especially because it is in the common scss and very generic mat-tooltip-component . mat-tooltip affecting everything in the project

Reply:
Resolved.